ha/util/conf_store.py

```diff
@@ -110,3 +110,111 @@ def set_cluster_cardinality(self, index):
         cluster_cardinality_key = CLUSTER_CARDINALITY_KEY
         cluster_cardinality_value = {CLUSTER_CARDINALITY_NUM_NODES: num_pods, CLUSTER_CARDINALITY_LIST_NODES : watch_pods }
         self._confstore.update(cluster_cardinality_key, json.dumps(cluster_cardinality_value))
+
+    @staticmethod
+    def _get_cvg_count(index, node_id):
+        cvg_count = Conf.get(index, GconfKeys.CVG_COUNT.value.format(_DELIM=_DELIM, node_id=node_id))
+        if not cvg_count:
+            Log.warn(f"CVGs are not available for this node {node_id}")
+        return cvg_count
+
+    @staticmethod
+    def get_cvg_list(index, node_id):
+        """
+        Return list of CVG's for any given node.
+        Args:
+            index(str): index of conf file
+            node_id (str): node id
+        Returns:
+            list: list of CVG's
+
+        >>> ConftStoreSearch.get_cvg_list("cortx", '21d6291109304485b3daff43a06cff77')
+        ['cvg-01', 'cvg-02']
+        """
+        cvg_list = []
+        try:
+            cvg_count = ConftStoreSearch._get_cvg_count(index, node_id)
+            if cvg_count:
+                cvg_list = [Conf.get(index, GconfKeys.CVG_NAME.value.format
+                                     (_DELIM=_DELIM, node_id=node_id, cvg_index=cvg_index)) for cvg_index in range(cvg_count)]
+            return cvg_list
+        except Exception as e:
+            Log.error(f"Unable to fetch CVG list from GConf. Error {e}")
+            raise Exception(f"Unable to fetch CVG list. Error {e}")
```

Comment:
Please confirm if raising exception is fine or this needs to be a error log.
Same comment for all functions